godot-macros/src/derive/data_models/unit_only_enum.rs

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use proc_macro2::{Ident, Literal, Span, TokenTree};
+
+use crate::util::{bail, error};
+use crate::ParseResult;
+
+/// Stores info from unit-only enums for use in deriving `GodotConvert` and other related traits.
+#[derive(Debug, Clone)]
```

Comment:
```suggestion
#[derive(Clone, Debug)]
```
😉 

Also some others.